cpp/src/traversal/bfs_kernels.cuh

```diff
@@ -353,7 +354,7 @@ __global__ void main_bottomup_kernel(const IndexType *unvisited,
 
       // If we haven't found a parent and there's more edge to check
       if (!found && degree > MAIN_BOTTOMUP_MAX_EDGES) {
-        left_unvisited_off = atomicAdd(left_unvisited_cnt, (IndexType)1);
+        left_unvisited_off = traversal::atomicAdd(left_unvisited_cnt, (IndexType)1);
```

Comment:
static_cast<IndexType>(1) instead of old C type cast.

Reply:
Done.